From 993b174fb0edf3d60d41c9aee74d15869edf2533 Mon Sep 17 00:00:00 2001
From: Hans Christian Lonstad <hcl@datarespons.no>
Date: Mon, 18 Nov 2019 15:40:58 +0100
Subject: [PATCH 4/6] Remove unconditional fix for ARM ERRATA 845369

This fix has severe performance implications for som memory intensive
applications (like Java VM garbage collector).
---
 arch/arm/mach-imx/mx6/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-imx/mx6/Kconfig b/arch/arm/mach-imx/mx6/Kconfig
index 3519cf6821..dfe9bc59ae 100644
--- a/arch/arm/mach-imx/mx6/Kconfig
+++ b/arch/arm/mach-imx/mx6/Kconfig
@@ -4,7 +4,6 @@ config MX6_SMP
 	select ARM_ERRATA_751472
 	select ARM_ERRATA_761320
 	select ARM_ERRATA_794072
-	select ARM_ERRATA_845369
 	select MP
 	bool
 
-- 
2.20.1

